gapil/template/constant_sets.go

```diff
@@ -175,6 +175,41 @@ func buildConstantSets(api *semantic.API, mappings *resolver.Mappings) *Constant
 
 	constsets := map[semantic.Node]*int{}
 
+	for _, e := range api.Enums {
+		if e.Annotations.GetAnnotation("analyze_usage") == nil {
+			labels := analysis.Labels{}
+			for _, entry := range e.Entries {
+				value := uint64(0)
+				switch v := entry.Value.(type) {
```

Comment:
This switch also [exists here](https://github.com/google/gapid/blob/master/gapil/analysis/value.go#L413). Please add a `func AsUint64(Expression) (uint64, bool)` function to `gapil/semantic/type.go` next to the other helpers (`IsInteger()`, `IsUnsigned()`, etc)